content/docs/04-ai-sdk-rsc/05-streaming-values.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: Streaming Values
+description: Overview of streaming RSCs
+---
+
+import { UIPreviewCard, Card } from '@/components/home/card';
+import { EventPlanning } from '@/components/home/event-planning';
+import { Searching } from '@/components/home/searching';
+import { Weather } from '@/components/home/weather';
+
+# Streaming Values
+
+The RSC API provides several utility functions to allow you to stream values from the server to the client. This is useful when you need more granular control over what you are streaming and how you are streaming it.
+
+<Note>
+  These utilities can also be paired with [AI SDK Core](/docs/ai-sdk-core)
+  functions like [`streamText`](/docs/reference/ai-sdk-core/stream-text) and
+  [`streamObject`](/docs/reference/ai-sdk-core/stream-object) to easily stream
+  LLM generations from the Server to the Client.
```

Comment:
q: we capitalize server and client here but not anywhere else, which do we prefer?

Reply:
I think lowercase